### ISSUE TITLE
Feature: Convert to ePUB and PDF for readers

### ISSUE THREAD

**Issue by elog08** [geezorg/ebooks#1]
Thanks for putting these out. Are there any plans to convert these to ePUB/PDF so they can be consumed by book reader apps?

Thanks

**Comment by dyacob** [comment]
yes, eventually the ebook formats are the end of the pipeline.  its a long process, following proofreading, formatting, layout, lots of clean up work.  help is certainly welcomed on these tasks :-)

**Comment by seid0119** [comment]
A
